core/trino-main/src/main/java/io/trino/server/remotetask/RequestErrorTracker.java

```diff
@@ -138,7 +140,9 @@ public void requestFailed(Throwable reason)
         }
     }
 
-    static void logError(Throwable t, String format, Object... args)
+    @FormatMethod
+    @SuppressWarnings("FormatStringAnnotation")
```

Comment:
Why this is needed?

Reply:
That's because of this line:

```java
             log.error(format + ": %s", ObjectArrays.concat(args, t));
```

It kind of appends the `Throwable` as the last thing after the original format string. I tried to make it acceptable to Error Prone, or at least make the suppression more local, but couldn't. If you can think of a better way (or perhaps to get rid of it entirely), I'd be glad to fix it properly :)

Reply:
Please add a comment. This is fine.